layouts/_default/baseof.html

```diff
@@ -80,8 +80,9 @@
       {{ end }}
     {{ end }}
 
-    <link rel="icon" type="image/png" href="{{ .Site.Params.favicon_32 | default "/images/favicon-32x32.png" | absURL }}" sizes="32x32">
-    <link rel="icon" type="image/png" href="{{ .Site.Params.favicon_16 | default "/images/favicon-16x16.png" | absURL }}" sizes="16x16">
+    {{ range .Site.Params.favicons }}
+      <link rel="{{ .rel }}" type="{{ .type }}" href="{{ .href | absURL }}" sizes="{{ .sizes }}">
```

Comment:
This change causes that favicons is a required config item? In my opinion the template should provide sensible defaults that can be overwritten with an optional parameter.

Reply:
It also introduces a **breaking change**, since the `favicons` is a new config parameter.

Reply:
How about default the initial two, else if custom settings are specified, my changes?